### ISSUE TITLE
quesion about the output

### ISSUE THREAD

**Issue by yechenzhi** [RLHFlow/RLHF-Reward-Modeling#11]
I'm using your llama3 script with deepspeed3 to train a reward model, and I set **save_every_steps** to 20. 

However I'm confused about those files in the output, for example in checkpoint-40, there are 4 safetensors: _model-00001-of-00004.safetensors, model-00002-of-00004.safetensors,model-00003-of-00004.safetensors,model-00004-of-00004.safetensors_. Are those safetensors the fine-tuned checkpoint?

If so, what is in the global_step40? there are some pt files include: _bf16_zero_pp_rank_0_mp_rank_00_optim_states.pt  bf16_zero_pp_rank_2_mp_rank_00_optim_states.pt  zero_pp_rank_0_mp_rank_00_model_states.pt  zero_pp_rank_2_mp_rank_00_model_states.pt
bf16_zero_pp_rank_1_mp_rank_00_optim_states.pt  bf16_zero_pp_rank_3_mp_rank_00_optim_states.pt  zero_pp_rank_1_mp_rank_00_model_states.pt  zero_pp_rank_3_mp_rank_00_model_states.pt._ 

Can I just delete those pt files as they take up a huge space.

**Comment by WeiXiongUST** [comment]
The file is very large because the trainer also saves the optimizer, which can be much larger than your model. One way is to load the model and save the model to another dir so only the model itself is saved 

```python

model = AutoModelForSequenceClassification.from_pretrained(
    old_dir, num_labels=1, torch_dtype=torch.bfloat16,
)
tokenizer = AutoTokenizer.from_pretrained("old_dir") 

model.save_pretrained("new_dir")
tokenizer.save_pretrained("new_dir")


```
